stingray/pulse/tests/test_pulse.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 from stingray.pulse.pulsar import fold_events, get_TOA, phase_exposure
 from stingray.pulse.pulsar import profile_stat, z_n, pulse_phase
+from stingray.pulse.pulsar import z_n, z_n_events, z_n_poisson, z_n_gauss, H
```

Comment:
So I'm not sure where it's defined, but is `H` a class or a function? I think our coding rules suggest that all functions should be lower-case, and I'm generally against single-letter function definitions. That seems like a really easy way to generate bugs. Can this be renamed to "htest" or something similar?

Reply:
Done